infra.bs

```diff
@@ -486,6 +486,24 @@ implementations of just <a>JavaScript strings</a> for performance and memory rea
 
 <hr>
 
+<p>To <dfn export for=string lt=concatenate|concatenation>concatenate</dfn> a <a for=/>list</a> of
+<a for=/>strings</a> <var>list</var>, with an optional separator string <var>separator</var>, run
+these steps:
+
+<ol>
+ <li><p>If <var>list</var> <a for=list>is empty</a>, then return the empty string.
+
+ <li><p>If <var>separator</var> is not given, then set <var>separator</var> to the empty string.
+
+ <li><p>Return a <a for=/>string</a> whose contents are <var>list</var>'s <a for=list>items</a>, in
+ order, separated from each other by <var>separator</var>.
+</ol>
+
+<p class=example id=example-string-concatenate>To serialize a set <var>set</var>, return the
+<a for=string>concatenation</a> of <var>set</var> with U+0020 SPACE.
```

Comment:
Here you are conflating a single code point and an entire string. Not sure whether we should be more careful, or just let it slide. (Maybe have a clause somewhere saying that single code points can be interpreted as strings when desired.)

Reply:
Yeah, I was wondering about that. I was hoping we could just cast between those when needed. Same with sets and lists and such.

Reply:
Happy to add text for this though.

Reply:
Seeing the usage sites it seems like a good idea to add, perhaps in a separate PR.

Reply:
Filed #188. Addressed the remainder.